src/IRCNetwork.cpp

```diff
@@ -425,6 +425,12 @@ bool CIRCNetwork::ParseConfig(CConfig* pConfig, CString& sError,
             CString sModName = sValue.Token(0);
             CString sNotice = "Loading network module [" + sModName + "]";
 
+            // We removed partyline on 2019-04-12, znc 1.8
+            if (sModName == "partyline") {
```

Comment:
This prevents even a 3rd party module from being loaded if they re-add it, they can work around this by naming it "partyline2" or whatever.

I considered not adding this block but I think it makes more sense to warn people than to give a generic `not found`. WDYT?